lib/decodeCalldata.test.ts

```diff
@@ -1,18 +1,22 @@
+import { BigNumber } from '@ethersproject/bignumber'
 import { expect } from 'earljs'
 
 import { decodeCalldata } from './decodeCalldata'
 import { parseAbi } from './parseAbi'
 
 describe(decodeCalldata.name, () => {
   it('handles expected case', () => {
-    const iface = parseAbi('function transferFrom(address,address,uint256),')!
+    const iface = parseAbi('function transferFrom(address,address,uint256)')!
     expect(iface).toBeDefined()
-    expect(
-      decodeCalldata(
-        iface,
-        '0x23b872dd0000000000000000000000008ba1f109551bd432803012645ac136ddd64dba72000000000000000000000000ab7c8803962c0f2f5bbbe3fa8bf41cd82aa1923c0000000000000000000000000000000000000000000000000de0b6b3a7640000',
-      ),
-    ).toBeAnObjectWith({ decoded: expect.defined(), fragment: expect.defined() })
+    const decoded = decodeCalldata(
+      iface,
+      '0x23b872dd0000000000000000000000008ba1f109551bd432803012645ac136ddd64dba72000000000000000000000000ab7c8803962c0f2f5bbbe3fa8bf41cd82aa1923c0000000000000000000000000000000000000000000000000de0b6b3a7640000',
+    )!
+    expect(decoded.decoded[0]).toEqual('0x8ba1f109551bD432803012645Ac136ddd64DBA72')
```

Comment:
this misses a check which function got decoded. You might want to use: `decoded.fragment.format()`